ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/HasRole.java

```diff
@@ -1,5 +1,6 @@
 package uk.gov.hmcts.ccd.sdk.api;
 
+@ComplexType(generate = false)
```

Comment:
do we need this annotation ?

Reply:
technically it doesn't need the generate = false but I left it there to indicate why I'd included the annotation. Now that the OrganisationPolicy<R extends HasRole> it started to create HasRole as a complex type. I needed to explicitly tell it not to